testData/unitData.ts

```diff
@@ -70,6 +77,23 @@ export const newService = async (): Promise<string> => {
     let n = faker.string.numeric(5)
     return `${searchService[0].toUpperCase()}${searchService.slice(1)} ${n}`;
 }
+export const adminCreds = {
+    email: "txt2021@ukr.net",
+    password: "Qwerty123+",
+    phone: "+380XXXXXXXXX",
+    
+}
```

Comment:
Use GitHub secrets for these creds:
```suggestion
export const adminCreds = {
    email: process.env.ADMIN_EMAIL,
    password: process.env.ADMIN_PASSWORD,
    phone: process.env.ADMIN_PHONE,
}
```